### PR TITLE
RMT Guide: integrate proofing corrections

### PR DESCRIPTION
**Pull request by dariavladykina**
### PR creator: Description

Integrate proofing corrections for the RMT Guide from proofing drop 3.

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP6/openSUSE Leap 15.6
  - [ ] SLE 15 SP5/openSUSE Leap 15.5
  - [ ] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2

- SLE 12
  - [ ] SLE 12 SP5

### CODE REVIEW

File: xml/rmt_config_files.xml [hunk 1]
```diff
@@ -93,7 +93,7 @@
         <para>
           A list of domains that should <emphasis>not</emphasis> go through the
           proxy, separated by commas. For example:
-          <literal>localhost,.mylocaldomain</literal>.
```
[the remark this answers is not in the document]
Comment (dariavladykina) [reply xml/rmt_config_files.xml:96]: Tomas has confirmed the correct spelling of` localhost.mylocaldomain` here.

[the remark this answers is not in the document]
Comment (taroth21) [reply xml/rmt_config_files.xml:96]: Ok, this thas been clarified with Tomas internally already and is correct as it is.

File: xml/rmt_migrate_from_smt.xml [hunk 2]
```diff
@@ -746,7 +746,7 @@
     <orderedlist>
       <listitem>
         <para>
-          Support via &sliberty;, find more details in <link
+          Support via &sliberty; &mdash; find more details in <link
```
Comment (dariavladykina) [inline xml/rmt_migrate_from_smt.xml:749]:
```suggestion
          Support via &sliberty;&mdash;find more details in <link
```

**Review by taroth21 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**